pkg/pipelines/image-scan.go

```diff
@@ -44,7 +44,8 @@ func NewImageScan(stdout io.Writer, stderr io.Writer) *ImageScan {
 		Stdout: stdout,
 		Stderr: stderr,
 		artifactConfig: ArtifactConfig{
-			Directory:     os.TempDir(),
+			// Directory:     os.TempDir(),
```

Comment:
This should default to a temporary directory unless the user specifies otherwise